README.md

```diff
@@ -1,82 +1,479 @@
-
-# What is a validator?
+# Tableland Validator
+
+[![Review](https://github.com/tablelandnetwork/go-tableland/actions/workflows/review.yml/badge.svg)](https://github.com/tablelandnetwork/go-tableland/actions/workflows/review.yml)
+[![Test](https://github.com/tablelandnetwork/go-tableland/actions/workflows/test.yml/badge.svg)](https://github.com/tablelandnetwork/go-tableland/actions/workflows/test.yml)
+[![Go Reference](https://pkg.go.dev/badge/github.com/textileio/go-tableland.svg)](https://pkg.go.dev/github.com/textileio/go-tableland)
+[![Go Report Card](https://goreportcard.com/badge/github.com/textileio/go-tableland)](https://goreportcard.com/report/github.com/textileio/go-tableland)
+[![License](https://img.shields.io/github/license/tablelandnetwork/go-tableland.svg)](./LICENSE)
+[![Release](https://img.shields.io/github/release/tablelandnetwork/go-tableland.svg)](https://github.com/tablelandnetwork/go-tableland/releases/latest)
+[![standard-readme compliant](https://img.shields.io/badge/standard--readme-OK-green.svg)](https://github.com/RichardLitt/standard-readme)
+
+> Go implementation of the Tableland database—run your own node, handling on-chain mutating events and serving read-queries.
+
+## Table of Contents
+
+- [Background](#background)
+  - [What is a validator?](#what-is-a-validator)
+  - [Validator and network relationship](#validator-and-network-relationship)
+  - [Running a validator](#running-a-validator)
+- [Usage](#usage)
+  - [System requirements](#system-requirements)
+  - [Firewall configuration](#firewall-configuration)
+  - [System prerequisites](#system-prerequisites)
+  - [Run the validator](#run-the-validator)
+  - [Docker Compose setup](#docker-compose-setup)
+  - [Backups and other routines](#backups-and-other-routines)
+- [Development](#development)
+  - [Configuration](#configuration)
+- [Contributing](#contributing)
+- [License](#license)
+
+## Background
+
+`go-tableland` is a Go language implementation of a Tableland node, enabling developers and service providers to run nodes on the Tableland network and host databases for web3 users and applications. Note that the Tableland protocol is currently in open beta, so node operators have the opportunity to be one of the early network adopters while the responsibilities of the validator will continue to change as the Tableland protocol evolves.
+
+### What is a validator?
 
 Validators are the execution unit/actors of the protocol.
 
 They have the following responsibilities:
 
 - Listen to on-chain events to materialize Tableland-compliant SQL queries in a database engine (currently, SQLite by default).
-- Serve read-queries (e.g: `SELECT * FROM foo_69_1`) to the external world.
+- Serve read-queries (e.g., `SELECT * FROM foo_69_1`) to the external world.
+
+> In the future, validators will have more responsibilities in the network.
+
+### Validator and network relationship
```

Comment:
i am wondering if we should call them a validator at the moment. or maybe just call them a "node"?

Reply:
umm i'll defer to you guys on that. but, we do call it a "validator" or "validator node" pretty much _everywhere_ in our docs, so we'd have to update that as well. plus, the SDK calls has a "Validator" API. i'm fine either way, just want it to be consistent!

Reply:
i mean, i don't have a strong opinion here too. it should be okay since we are mentioning that the nodes will be _validating_   in the future